week-03/assessment/solar-farm/solar-farm/data/log-entries.csv

```diff
@@ -0,0 +1,5 @@
+id,section,row,column,yearinstalled,material,istracking
```

Comment:
You can delete either log-entries.csv or solarpanelfiles.csv - both files are not needed